### PR TITLE
misc: add __init__.py to api directory

### PR DESCRIPTION
**Pull request by roks0n**
## Proposed changes

Not having `__init__.py` in the api directory results in api directory to not be included in the package. I've tested this locally as well by building it and noticed the api directory was present after this change.

Also bumping the client version so it can get built and uploaded to pypi.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [x] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/developers/guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

None

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,3 +9,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## 0.1.0 - 2018-08-27
 - Initial Release
+
+## 0.1.01 - 2018-08-28
```
Comment (faustbrian) [inline CHANGELOG.md]: This goes above the `0.1.0` release and should be `0.1.1`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,3 +9,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Added `__init__.py` to the api directory in order to include the api into the package
```
Comment (faustbrian) [inline CHANGELOG.md]: Should be grouped under `### Fixed`

File: setup.py [hunk 3]
```diff
@@ -28,7 +28,7 @@
 setuptools.setup(
     name='arkecosystem-client',
     description='A simple Python API client for the ARK Blockchain.',
-    version='0.1.0',
+    version='0.1.01',
```
Comment (faustbrian) [inline setup.py]: Should be `0.1.1`

**Comment by roks0n** [comment]
@faustbrian done, please confirm if I've done the changelog correctly - I'm not 100% sure if I did.

**Comment by faustbrian** [comment]
You can always check https://keepachangelog.com/en/1.0.0/ if you have any doubt about the changelog format.

**Pull request merged**